scalafmt-tests/src/test/scala/org/scalafmt/stats/GitInfo.scala

```diff
@@ -2,7 +2,7 @@ package org.scalafmt.stats
 
 import scala.util.Try
 
-// Name and email is anyways registered in the git log.
+// Name and email is registered in the git log.
```

Comment:
`anyways` is colloquially acceptable, but it's generally awkward and I don't think it adds any value here.

Reply:
then drop `s` and keep `anyway`. can move to after `registered` if you'd like. let's fix typos and preserve the creative genius behind the original comment :)